guides/v2.2/frontend-dev-guide/css-topics/css-preprocess.md

```diff
@@ -140,6 +140,8 @@ You can find the detailed information about the configuration and other options
 
 In client-side compilation mode, most of the stylesheet customizations display immediately after you reload a page in a browser.
 
+Working in LESS files are compiled every page load on the client-side compilation mode. This results in exceptionally slow response times and horrible flash-of-unstyled-text.
```

Comment:
Please, rephrase it slightly to have something like the following description in result:

"With client-side compilation mode enabled LESS files are compiled on every page load. This results in slow page loading performance.". 

And remove the "horrible flash-of-unstyled-text"

Thank you!

Reply:
@rogyar I have updated requested changes please review.